README.md

```diff
@@ -1,5 +1,176 @@
+| Version   | 0.6.0                                         |
+| Source    | https://github.com/VNG-Realisatie/zgw-infra   |
+| Keywords  | ZGW, kubernetes, helm                         |
+| Related   | https://github.com/VNG-Realisatie/gemma-zaken |
 
-This repo is meant for storage of all things Referentie implementatie that relate to Infra and Deployment.
\ No newline at end of file
+
+## Introductie
+
+Om het zaakgericht werken in de Common Ground architectuuur te ondersteunen zijn de [API's voor Zaakgericht werken ontwikkeld](https://github.com/VNG-Realisatie/gemma-zaken). 
+
+## Aan de slag
+
+De eerste stap om de helm chart te draaien is uiteraard het opzetten van een kubernetes cluster en een lokale versie van helm.
+Het is aan de raden een helm versie van `>= 3.8` te nemen omdat deze het toestaat helm charts als oci images te downloaden.
+
+Voor helm zie:
+
+https://github.com/helm/helm/releases
+https://helm.sh/docs/intro/install/
+
+Hoe je kubernetes lokaal draait hangt af van je systeem maar gemakshalve worden hier alleen `minikube` en `docker-desktop` behandeld.
+
+### docker-desktop
+
+Docker-desktop is de standaard om kubernetes te draaien op MacOs en Windwos.
+Doordat docker-desktop een gui heeft is het voldoende docker-desktop te starten.
+
+Docker-desktop heeft geen eigen ingress dus deze moet je instellen. Hiervoor zit in de `Makefile` een commando:
+
+```shell
+make -C helm/ri nginx_install 
+```
+
+### minikube
+
+```shell
+minikube start --cpus 2 --memory 4000 --driver=hyperkit --container-runtime=docker
+minikube addons enable ingress
+minikube ip
+eval $(minikube docker-env)
+```
+
+Deze commando's starten een minikube cluster met docker en hyperkit. Er zijn veel varaianten mogelijk dus dit dient louter als voorbeeld.
+Zie bijvoorbeeld [minkube](https://minikube.sigs.k8s.io/docs/start/).
+
+Zorg dat je ingress activeert en het ip-adres van minikube opvraagt (minikube draait immers in een virtuele machine met een eigen IP).
+Het IP heb je nodig in de ingress stap.
+
+Mocht je lokaal images willen gebruiken het laatste commando.
+
+Hierna kun je aan de slag!
+
+## Namespace
+
+Voor je de helm chart installeert is het nodig een namespace te maken. Deze staat standaard op `vng` maar is aan te passen in de `values.yaml`.
+
+Onderstaand commando gaat uit van de standaard:
+
+```shell
+kubectl create ns vng
+```
+
+## Parser
+
+De parser is een python script die voor lokaal gebruik een config opbouwt in de `values.yaml`.
+De `env.yaml` wordt gebruikt om een configuratie te laden. Standaard wordt `local` genomen maar deze is eenvoudig te overschrijven door:
+
+```shell
+export ENV=production
+```
+
+te zetten. Of eventueel mee te geven in PyCharm. Daarna is het eenvoudig een kwestie van:
+
+```shell
+cd ./parser
+pip install -r requirements.txt
+python3 parser.py
+```
+
+## Lokaal
+
+Als je een nieuwe config hebt gemaakt met de parser kun je de helm chart installeren.
+
```

Comment:
'ls' command geeft:
Chart.yaml  Makefile    charts      templates   values.yaml
Vervolgens : helm install ri .
Error: INSTALLATION FAILED: error unpacking charts in ri-zgw: Chart.yaml file is missing

Doe ik iets verkeerd?

Reply:
Ja

Reply:
Nee probeer het na een pull en fetch nog eens.